Snowflake.Data.Tests/SFConnectionPoolT.cs

```diff
@@ -38,6 +45,7 @@ public void TestBasicConnectionPool()
         }
 
         [Test]
+        [Ignore("Disable unstable test cases for now")]
```

Comment:
Why disabling tests?

Reply:
@sfc-gh-hyu Those test failed time to time and I don't think it's related to my changes.
It only occurs when running test on github action and I can't reproduce it on my local. I've made a test PR https://github.com/snowflakedb/snowflake-connector-net/pull/478 using the same code as master branch and I can see those test failures on that PR. I also see those failures on Another unrelated PR https://github.com/snowflakedb/snowflake-connector-net/pull/480 as well.
I feel those failures might related to connection pooling implementation and will continue to look into the root cause. For now since it's not related to my change I disable those test case to pass the test.

Reply:
OK, I am fine with disabling them for now, but please make sure that re-enabling those tests should be top priority.